vcstool/clients/git.py

```diff
@@ -537,6 +537,8 @@ def log(self, command):
                 return result_tag
             # output log since nearest tag
             cmd = [GitClient._executable, 'log', '%s..' % result_tag['output']]
+        elif command.merge_only:
```

Comment:
The logic for this new options seems to be mutually exclusive to the other options (e.g. `limit-tag`, `limit-untagged`). But I don't see a reason why these couldn't be combined / passes at the same time?

Reply:
Makes sense. Updated.